arangod/RestServer/FileDescriptorsFeature.cpp

```diff
@@ -59,7 +60,7 @@ struct FileDescriptors {
     if (res != 0) {
       LOG_TOPIC("17d7b", FATAL, arangodb::Logger::SYSCALL)
           << "cannot get the file descriptors limit value: " << strerror(errno);
-      FATAL_ERROR_EXIT();
+     FATAL_ERROR_EXIT_CODE(TRI_EXIT_RESOURCES_TO_LOW);
```

Comment:
This is actually not the correct place for setting this error code.
Here we are just _querying_ the current limits, but not adjusting them.

Adjustment happens in the `store` function.